Sources/DistributedActors/ClusterSystem.swift

```diff
@@ -986,7 +986,7 @@ extension ClusterSystem {
 // MARK: Intercepting calls
 
 extension ClusterSystem {
-    public func interceptCalls<Act, Interceptor>(
+    internal func interceptCalls<Act, Interceptor>(
```

Comment:
great, thanks -- I was a bit nervous if it is the right exact API we want to support forever.

I'm pretty sure we'll expose this eventually, but let's buy us some time :)